cli/src/pcluster/cli/commands/common.py

```diff
@@ -55,8 +60,57 @@ def execute(self, args, extra_args) -> None:
         """Execute CLI command."""
         pass
 
+    @staticmethod
+    def _exit_on_http_status(http_response):
+        if 200 <= http_response.status_code <= 299:
+            sys.exit(0)
+        else:
+            sys.exit(1)
+
+
+class CliCommandV3(CliCommand, ABC):  # TODO: remove once all commands are converted
+    """Temporary class to identify pcluster v3 commands."""
+
+    pass
+
 
 class DcvCommand(CliCommand, ABC):
     """Abstract class for DCV CLI commands."""
 
     pass
+
+
+def print_json(obj):
+    """Print formatted Json to stdout."""
+    print(json.dumps(obj, indent=2))
+
+
+def csv_type(choices):
+    """Return a function that splits and checks comma-separated values."""
+
+    def split_arg(arg):
+        values = set(token.strip() for token in arg.split(","))
+        for value in values:
+            if value not in choices:
+                raise argparse.ArgumentTypeError(
+                    "invalid choice: {!r} (choose from {})".format(value, ", ".join(map(repr, choices)))
```

Comment:
Where is `repr` coming from?

Reply:
`map(function, iterable, ...)` applies `function` to each value of the iterable collection. in this case it calls repr (the built-in python function) to get a printable representation of the object